exporter/awsxrayexporter/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/exporter/awsxra
 go 1.14
 
 require (
-	github.com/aws/aws-sdk-go v1.35.17
+	github.com/aws/aws-sdk-go v1.35.20
```

Comment:
Doesn't it belong to another PR?

Reply:
`make gotidy` created them :)). Let me rebase.

Reply:
All unexpected go.mod changes are gone.